engine-tests/src/tests/standard_precompiles.rs

```diff
@@ -60,25 +60,25 @@ fn profile_modexp() {
 #[test]
 fn profile_ecadd() {
     let profile = precompile_execution_profile("test_ecadd");
-    utils::assert_gas_bound(profile.all_gas(), 6);
+    utils::assert_gas_bound(profile.all_gas(), 5);
 }
 
 #[test]
 fn profile_ecmul() {
     let profile = precompile_execution_profile("test_ecmul");
-    utils::assert_gas_bound(profile.all_gas(), 7);
+    utils::assert_gas_bound(profile.all_gas(), 6);
 }
 
 #[test]
 fn profile_ecpair() {
     let profile = precompile_execution_profile("test_ecpair");
-    utils::assert_gas_bound(profile.all_gas(), 102);
+    utils::assert_gas_bound(profile.all_gas(), 115);
```

Comment:
A 10% increase in the gas cost here is a little concerning. Is the new rust compiler worse at optimizing things in some cases now? Or is it something to do with the Near Runtime that is different now?

Reply:
Good question. I'm not sure that we can compile the code with the previous Rust compiler since there are a lot of updated crates which don't support it.